### DISCUSSION TITLE
How to avoid sector number conflict when missing some backup.

### DISCUSSION THREAD

**Discussion by de-authority** [filecoin-project/lotus#7179] [Ideas]
I have heard more than one community member feedback that when they restore miner, they sealing new sectors during the time between the backup and the problem, and when they restore miner, there is no backup in the middle part of the time, and when they start sealing sectors again, the sector number will start from the backup, which causes a sector number conflict, and they have to manually terminate this part of the sectors, which causes penalty. I want to know if it is possible to avoid this situation, or how to reduce the loss if this situation occurs?  Some people say we can modify leveldb.

**Comment by de-authority** [comment]
there is one of our Chinese members describe: "23号系统盘坏了，miner启动打印leveldb损坏，然后把20号备份的.lotusminer数据恢复，使用20号.lotusminer启动成功，24号的post也能顺利通过；但是25号开始使用lotus-miner sectors pledge xxx进行质押（发现质押的扇区号是20号的起始扇区号），且发现只会进行ap阶段，p1\p2都不会进行，miner此时会打印remove 20号的扇区，然后就发现存储的20号已封好的扇区被删除"

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
you can manipulate the db.  search for a post in slack by @karalabe. he did open heart surgery on it before.

you can try:

- disconnect all "store" storage from the miner
- shut down all workers
- pledge sectors

then the number should be counted up while nothing happens. do this at your own risk. there might be more chaos following.... not sure.

see these issues before thinking about trying it:

https://github.com/filecoin-project/lotus/issues/5358
https://github.com/filecoin-project/lotus/issues/5354

**Reply by de-authority** [reply]
Thanks patrick， why your issue get no answer?  it is not in high priority?
